ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```
@@ -846,6 +846,20 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-winc
+- as: aws-ipi-ovn-efs-hypershift-mgmt-longduration-f365-destructive
+  cron: 13 18 4 * *
```

```suggestion
- as: aws-ipi-ovn-efs-hypershift-mgmt-longduration-f999-destructive
  cron: 8 8 29 2 *
```
what @heliubj18 means reduce the frequency should also make the frequency `f-` match the cron

Please use tool generate the cron
```console
$ ./ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh aws-ipi-ovn-efs-hypershift-mgmt-longduration-f999-destructive openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml
cron: 8 8 29 2 *
```